Heart/Team/Presentation/Requests/CreateTeamRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Heart\Team\Presentation\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateTeamRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            'leader_id' => [
+                'required', 'exists:users,id', 'unique:teams,leader_id'
```

Comment:
Esse requisito está correto?

1 membro só pode ser Líder de 1 time? Se esse for o caso, recomendo fazer em outra camada.

1. Erro de responsa confuso ou indireto
2. A mesma lógica se aplica ao Sub-lider provavelmente que não está nessa requisição.
3. Menor manutabilidade